build.gradle.kts

```diff
@@ -47,7 +47,12 @@ dependencies {
 }
 
 group = "org.opensearch"
-version = "2.0.0-SNAPSHOT"
+var isSnapshot = "true" == System.getProperty("build.snapshot", "true")
+var version_temp = "2.0.0"
```

Comment:
`version_temp` is probably not a good name, probably sthould be taken from `System.getProperty("build.version", "2.0.0")` property or property file (so we could do releases by providing the values in command line).

Reply:
We will change this var to either coming from env_var or coming from versions.properties later.